src/test/auth.spec.js

```diff
@@ -0,0 +1,67 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+import TestDatabase from '../utils/testDatabase';
+import Helper from '../utils/helpers';
+
+chai.use(chaiHttp);
+chai.should();
+
+const apiEndpoint = '/api/v1/auth/login';
+const user = {
+  email: 'user1@gmail.com',
+  password: '123456'
+};
+describe('/api/v1/auth', () => {
+  let verifiedUser, notVerifiedUser;
+
+  before((done) => {
+    TestDatabase.destroyUsers();
+    done();
+  });
+  describe('POST /login', () => {
+    before(async () => {
+      verifiedUser = await TestDatabase.createUser({
+        ...user,
+        is_verified: true,
+      });
+      notVerifiedUser = await TestDatabase.createUser({
+        ...user,
+        email: 'user2@gmail.com',
+      });
+    });
+    it('should return 400 if the user is not found', async () => {
+      const res = await chai.request(app)
+        .post(apiEndpoint)
+        .set('Content-Type', 'application/json')
+        .send({ email: 'email@email.com', password: 'password' });
+      res.should.have.status(400);
```

Comment:
@SamuelKoroh looks good. However, http status code for 'not found' should be 404 not 400. Please adjust this